Gopkg.toml

```diff
@@ -21,35 +21,37 @@ required = [ "k8s.io/code-generator/cmd/client-gen", "k8s.io/gengo/types" ]
 
 [[constraint]]
   name = "github.com/google/uuid"
-  version = "0.2.0"
+  revision = "8c31c18f31ede9fc8eae72290a7e7a8064e9b3e3"
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.11.1"
+  version = "kubernetes-1.12.0"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "8.0.0"
+  version = "9.0.0"
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.11.1"
+  # use origin fork with nullable support. We can switch back to the upstream version in 1.14.
```

Comment:
@sttts you need to sign your name and sig-master to this 

`# sttts, sig-master`

so they can chase us down if they need to.  I've been forcing other teams to do the same when mess about in our pond.

Reply:
fixed